apps/visualization/utils.py

```diff
@@ -33,17 +30,25 @@
 GLOBAL_LEVEL = 'global_level'
 
 
+def clean_filters(filters):
+    # Filter out None values
+    return {
+        k: v
+        for k, v in filters.items()
+        if v is not None
+    }
+
+
```

Comment:
Lets move this util to utils.py

Reply:
it is actually on visualization/utils.py, Do you mean to move it into config/utils.py ?

Reply:
yes